From 22a1a49e1d5fecaa4ba6cee5d9ffba13fb1176fc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=A0=D1=83=D1=81=D0=BB=D0=B0=D0=BD=20=D0=98=D0=B6=D0=B1?=
 =?UTF-8?q?=D1=83=D0=BB=D0=B0=D1=82=D0=BE=D0=B2?= <lrn1986@gmail.com>
Date: Mon, 13 Apr 2015 13:52:29 +0000
Subject: [PATCH 10/20] [ANGLE][W32] Don't hardcode ossource_posix.cpp in the
 makefile

Compile ossource_posix.cpp or ossource_win.cpp depending on the target.

Otherwise you get an error:
../webkitgtk-2.4.8/Source/ThirdParty/ANGLE/src/compiler/ossource_posix.cpp:13:2: error: #error Trying to build a posix specific file in a non-posix build.
---
 Source/ThirdParty/ANGLE/GNUmakefile.am | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Source/ThirdParty/ANGLE/GNUmakefile.am b/Source/ThirdParty/ANGLE/GNUmakefile.am
index 4c14ac1..70904bb 100644
--- a/Source/ThirdParty/ANGLE/GNUmakefile.am
+++ b/Source/ThirdParty/ANGLE/GNUmakefile.am
@@ -80,7 +80,6 @@ libANGLE_la_SOURCES = \
 	Source/ThirdParty/ANGLE/src/compiler/MMap.h \
 	Source/ThirdParty/ANGLE/src/compiler/NodeSearch.h \
 	Source/ThirdParty/ANGLE/src/compiler/osinclude.h \
-	Source/ThirdParty/ANGLE/src/compiler/ossource_posix.cpp \
 	Source/ThirdParty/ANGLE/src/compiler/OutputESSL.cpp \
 	Source/ThirdParty/ANGLE/src/compiler/OutputESSL.h \
 	Source/ThirdParty/ANGLE/src/compiler/OutputGLSLBase.cpp \
@@ -161,3 +160,9 @@ libANGLE_la_SOURCES = \
 	Source/ThirdParty/ANGLE/src/compiler/VersionGLSL.h \
 	Source/ThirdParty/ANGLE/src/third_party/compiler/ArrayBoundsClamper.cpp \
 	Source/ThirdParty/ANGLE/src/third_party/compiler/ArrayBoundsClamper.h
+
+if TARGET_WIN32
+libANGLE_la_SOURCES += Source/ThirdParty/ANGLE/src/compiler/ossource_win.cpp
+else
+libANGLE_la_SOURCES += Source/ThirdParty/ANGLE/src/compiler/ossource_posix.cpp
+endif
-- 
1.8.5.3

